ko.json

```
@@ -0,0 +1,142 @@
+    "footer_dmca": "DMCA (디지털 밀레니엄 저작권법)",
+    "footer_documents": "문서",
+    "footer_github": "Github",
+    "footer_discord": "디스코드",
+    "footer_info/status": "정보/상태",
+
+
+    "not_found": "404 에러 (파일이 존재하지 않습니다! 🙀)",
+    "not_found_message": "찾으시려는 항목이 존재하지 않습니다. 😿",
+    "not_found_message2": "더 이상 항목이 존재하지 않습니다. 메인 페이지로 돌아가는 것은 어떨까요? 😽",
+    "return_to_main_page": "메인 페이지로 돌아가기",
+
+    "oops": "어어...잠시만요!",
+    "search_results_do_not_exist": "고양이들이 열심히 1TB 이상의 맵셋들을 찾았지만 당신이 입력한 키워드의 맵셋을 찾을 수 없었습니다. 😥",
+
+    "loading": "로딩 중...",
+    "loaded": "로딩 완료!",
+
+    "announcement": "공지",
+
+    "please_wait_to_download": "다운로드 대기 중...",
+    "not_selected_to_download": "다운로드 하도록 선택되지 않음",
+    "zip_system_announcement": "압축 시스템 공지",
+    "zip_system_announcement_message1": "맵셋이 압축되었고, 다운로드되었습니다.",
+    "zip_system_announcement_message2": "맵셋 압축이 진행중입니다! 잠시만 기다려주세요.",
+    "zip_system_announcement_message3": "이 메세지는 {}초 후에 닫힙니다.",
+    "ok": "확인",
+
+    "thank_you_for_using_nerinyan": "Nerinyan을 이용해 주셔서 감사합니다! 🐈",
+    "your_requested_beatmapset_will_download": "선택하신 비트맵이 곧 다운로드 됩니다. 잠시만 기다리시면 리다이렉트 됩니다! 다운로드 시작이 안 되었나요?",
+    "click_here_to_download_manually": "여기를 눌러 다시 시도해 주세요.",
+
+
+    "search": "검색",
+    "search_placeholder": "키워드를 입력해주세요 😺",
+    
+    "mode": "모드",
+    "mode_any": "모두",
+    "mode_osu": "osu!",
+    "mode_taiko": "osu!taiko",
+    "mode_catch": "osu!catch",
+    "mode_mania": "osu!mania",
+
+    "categories": "카테고리",
+    "categories_any": "모두",
+    "categories_has_leaderboard": "리더보드 있음",
+    "categories_ranked": "랭크",
+    "categories_qualified": "퀄리파이",
+    "categories_loved": "러브드",
+    "categories_pending": "대기 중",
+    "categories_wip": "작업 중",
+    "categories_graveyard": "무덤에 감",
+
+    "explicit_content": "노골적인 컨텐츠",
+    "explicit_content_hide": "숨김",
+    "explicit_content_show": "표시",
+
+    "extra": "추가 옵션",
+    "extra_has_video": "비디오 포함",
+    "extra_has_storyboard": "스토리보드 포함",
+
+    "sort_by": "정렬 옵션",
+    "sort_by_title": "제목",
+    "sort_by_artist": "아티스트",
+    "sort_by_difficulty": "난이도",
+    "sort_by_ranked": "랭크 순",
+    "sort_by_updated": "업데이트 순",
+    "sort_by_plays": "플레이 횟수 순",
+    "sort_by_favorites": "즐겨찾기 많은 순",
+
+    "filter_ar": "AR",
+    "filter_cs": "CS",
+    "filter_od": "OD",
+    "filter_hp": "HP",
+    "filter_bpm": "BPM",
+    "filter_star_rating": "스타 레이팅",
+
+    "download_options": "다운로드 옵션",
+    "download_options_direct_download": "리다이렉트 무시",
+    "download_options_no_video": "비디오 미포함",
+    "download_options_no_background": "백그라운드 이미지 삭제",
+    "download_options_no_hitsound": "커스텀 히트사운드 삭제",
+    "download_options_no_storyboard": "스토리보드 제거",
+
+    "zip_download": "압축 후 다운로드 (베타)",
+    "zip_download_selected_download": "선택된 맵셋 압축",
+    "zip_download_all_download": "전체 맵셋 압축",
+
+    "show_beatmap_info": "비트맵 정보 표시",
+    "hide_beatmap_info": "바트맵 정보 숨기기",
+
+    "ranked": "랭크",
```

이거는 공식 웹페이지 번역 프로젝트에서도 고민되는 사항이라 
1. 랭크됨을 하게된다면 밑에 퀄파랑 럽드도 됨을 붙여야 됨 (통일성)
2. 그렇다고 다 떼면 랭크 의 의미가 바뀜
그래서 일단은 랭크만 랭크됨을 하도록 하겠습니다.
```suggestion
    "ranked": "랭크됨",
```